src/core/lib/experiments/config.h

```diff
@@ -31,7 +31,7 @@ namespace grpc_core {
 struct ExperimentMetadata {
   const char* name;
   const char* description;
-  const char* additional_constaints;
```

Comment:
I think this one is going to cause a bunch of grief when we import it internally - can we exclude it from this change and do it separately?

Reply:
Sure thing.